README.md

```diff
@@ -2,6 +2,8 @@
 
 [![Solidity](https://github.com/taikochain/taiko-mono/actions/workflows/solidity.yml/badge.svg)](https://github.com/taikochain/taiko-mono/actions/workflows/solidity.yml)
 
+[![codecov](https://codecov.io/gh/taikochain/taiko-mono/branch/main/graph/badge.svg?token=E468X2PTJC)](https://codecov.io/gh/taikochain/taiko-mono)
```

Comment:
Shall we use different badges for different flags ( one for contract and another one for relayer )? https://community.codecov.com/t/badges-for-flags/270/3

Reply:
yes, lets! labels `Relayer` and `Protocol `?

Reply:
Sounds good